packages/sources/css/src/components/card/src/index.css

```diff
@@ -122,13 +119,13 @@
   height: auto;
   border-radius: 8px;
   color: var(--vtmn-semantic-color_content-primary-reversed);
-  background-color: linear-gradient(
+  background: linear-gradient(
     180deg,
     rgb(0 16 24 / 0%) 20%,
     var(--vtmn-semantic-color_background-primary-reversed) 100%
   );
 }
 
 .vtmn-card_variant--full-image > .vtmn-card_content--opaque {
-  background-color: rgb(0 16 24 / 60%);
+  background: rgb(0 16 24 / 60%);
```

Comment:
why not keeping `background-color` in this case?

Reply:
nope it doesn't work with the second background-color, kind of bug I think.

Reply:
Ok, after research, it seems like we must set a linear-gradient with background, not background-color : https://developer.mozilla.org/fr/docs/Web/CSS/gradient/linear-gradient()